conf/routes

```diff
@@ -45,39 +45,45 @@ GET            /trustedEntity/profile             @com.yetu.oauth2provider.contr
-GET            /reset                             @com.yetu.oauth2provider.controllers.authentication.PasswordReset.startResetPassword
-POST           /reset                             @com.yetu.oauth2provider.controllers.authentication.PasswordReset.handleStartResetPassword
-GET            /reset/:mailToken                  @com.yetu.oauth2provider.controllers.authentication.PasswordReset.resetPassword(mailToken)
-POST           /reset/:mailToken                  @com.yetu.oauth2provider.controllers.authentication.PasswordReset.handleResetPassword(mailToken)
+GET            /reset                               @com.yetu.oauth2provider.controllers.authentication.PasswordReset.startResetPassword
+POST           /reset                               @com.yetu.oauth2provider.controllers.authentication.PasswordReset.handleStartResetPassword
+GET            /reset/:mailToken                    @com.yetu.oauth2provider.controllers.authentication.PasswordReset.resetPassword(mailToken)
+POST           /reset/:mailToken                    @com.yetu.oauth2provider.controllers.authentication.PasswordReset.handleResetPassword(mailToken)
 
 # Change Password
-GET            /password                          @com.yetu.oauth2provider.controllers.authentication.PasswordChange.page
-POST           /password                          @com.yetu.oauth2provider.controllers.authentication.PasswordChange.handlePasswordChange
+GET            /password                            @com.yetu.oauth2provider.controllers.authentication.PasswordChange.page
+POST           /password                            @com.yetu.oauth2provider.controllers.authentication.PasswordChange.handlePasswordChange
 
 
 # Authentication entry points for all providers
-GET            /authenticate/:provider            @com.yetu.oauth2provider.controllers.authentication.ProviderController.authenticate(provider, redirectTo: Option[String])
-POST           /authenticate/:provider            @com.yetu.oauth2provider.controllers.authentication.ProviderController.authenticateByPost(provider, redirectTo: Option[String])
+GET            /authenticate/:provider              @com.yetu.oauth2provider.controllers.authentication.ProviderController.authenticate(provider, redirectTo: Option[String])
+POST           /authenticate/:provider              @com.yetu.oauth2provider.controllers.authentication.ProviderController.authenticateByPost(provider, redirectTo: Option[String])
 
 
-POST           /api/authenticate/:provider        @com.yetu.oauth2provider.controllers.authentication.LoginApi.authenticate(provider, builder = "token")
+POST           /api/authenticate/:provider          @com.yetu.oauth2provider.controllers.authentication.LoginApi.authenticate(provider, builder = "token")
 
 #####################
 ##    setup      ####
 #####################
 
+# New registration page
+GET            /setup/registration                  com.yetu.oauth2provider.controllers.setup.SetupPage.registration
```

Comment:
these pages are okay for testing purposes locally, but should not be put live on the server until there is some functionality. At the moment they don't "do" anything yet.